src/components/Quiz/CurrentQuestion.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { useQuiz } from '../../context/quizContext';
+import { Quiz } from '../../database/database.type';
+import "./CurrentQuestion.css";
+
+type Prop={
+    currentQuiz:Quiz
+}
+
+export const CurrentQuestion=({currentQuiz}:Prop)=>{
+    const navigate = useNavigate();
+	const { quizState, quizDispatch } = useQuiz();
+	const [disableButtons, setDisableButtons] = useState<boolean>(false);
+	const [optionId, setOptionId] = useState<string>('');
```

Comment:
In useState, we don't need to give types. It already gets defined on giving initial value. If I am wrong please let me know.